src/Bundle/DependencyInjection/Compiler/CommandHandlersCompilerPass.php

```diff
@@ -25,4 +38,39 @@ public function process(ContainerBuilder $container)
 
         $commandCollector->addMethodCall('processCommandServices', [$commands]);
     }
+
+    /**
+     * @param ContainerBuilder $container
+     * @throws CommandFormTypeDuplicate
+     * @throws CommandFormTypeMissingFormTypeTag
+     */
+    private function processFormTypes(ContainerBuilder $container)
+    {
+        if (!$container->hasDefinition('command_bus_console.command_form_type_map')) {
+            return;
+        }
+
+        $commandFormTypesTags = $container->findTaggedServiceIds('command_form_type');
+        $commandFormTypesCollector = $container->getDefinition('command_bus_console.command_form_type_map');
+
+        $commandFormTypes = [];
+
+        foreach ($commandFormTypesTags as $serviceName => $service) {
+            $allTags = $container->getDefinition($serviceName)->getTags();
+
+            if (!array_key_exists('form.type', $allTags)) {
+                throw new CommandFormTypeMissingFormTypeTag();
+            }
+
+            foreach ($allTags['command_form_type'] as $commandFormTypeTag) {
```

Comment:
Is there a possibility to have multiple `command_form_type` tags on single service?